avaframe/runCom1DFA.py

```diff
@@ -1,30 +1,100 @@
 """
-    Run script for running python DFA kernel
+    Run script for running python com1DFA kernel
 """
+# Load modules
+# importing general python modules
+import time
+import argparse
+import pathlib
+import sys
+
 
 # Local imports
 import avaframe.in3Utils.initializeProject as initProj
 from avaframe.com1DFA import com1DFA
 from avaframe.in3Utils import cfgUtils
 from avaframe.in3Utils import logUtils
+from avaframe.in3Utils import fileHandlerUtils as fU
+
+
+def runCom1DFA(avalancheDir='', calibration='standard'):
+    """ Run com1DFA in the default configuration with only an
```

Comment:
add and optional calibration?

Reply:
changed to empty. 
What do you mean by optional?